packages/theme/src/types.ts

```diff
@@ -43,7 +43,10 @@ type PaletteAdditions = {
   linkHover: string;
   link: string;
   gold: string;
-  sidebar: string;
+  sidebar: {
+    backgroundColor: string;
```

Comment:
Since these properties are all within a palette it's implied that they are colors, so can just call it `background` etc.

Reply:
Good point, agreed. I erred on the side of being explicit but that makes more sense. Updated 👍